src/core/Table/TableRowMemoized.tsx

```diff
@@ -42,18 +46,21 @@ const TableRow = <T extends Record<string, unknown>>(props: {
   });
 
   return (
-    <div {...rowProps} key={rowProps.key} ref={rowRef}>
-      {row.cells.map((cell) => {
-        const cellProps = cell.getCellProps({
-          className: cx('iui-tables-cell', cell.column.cellClassName),
-          style: getCellStyle(cell.column),
-        });
-        return (
-          <div {...cellProps} key={cellProps.key}>
-            {cell.render('Cell')}
-          </div>
-        );
-      })}
+    <div className='iui-tables-rowgroup'>
+      <div {...rowProps} key={rowProps.key} ref={rowRef}>
+        {row.cells.map((cell) => {
+          const cellProps = cell.getCellProps({
+            className: cx('iui-tables-cell', cell.column.cellClassName),
+            style: getCellStyle(cell.column),
+          });
+          return (
+            <div {...cellProps} key={cellProps.key}>
+              {cell.render('Cell')}
+            </div>
+          );
+        })}
+      </div>
+      {isExpanded && expandedSubComponent?.(row)}
```

Comment:
I think we might want to arrange spacing ourselves.
![image](https://user-images.githubusercontent.com/81580355/117617201-de5ea780-b174-11eb-9b55-b2792061d584.png)

And somehow I feel that content should start with the 2nd column (I mean not to be aligned with expander column)

@FlyersPh9 Could you check up this PR from visual aspect?

Reply:
This looks really good, I'm happy you used the borderless button for the expand / collapse arrow.

Question about the row height: is the borderless button causing the rows height to be taller than it would be if it were just text?  If yes, can we adjust the padding so that the row height is close to / the same height as a table row without a button in it?

Aside from that, gets my 👍.

Reply:
@FlyersPh9  And what about padding inside the expanded content? Shouldnt it have some or we leave that freely for the user?

Reply:
Can you place 1 line of text within the expanded section and try to get the height of that to match the row above?

Reply:
@FlyersPh9  Yes, it looks like the button is stretching the row a bit since small icon button has fixed height of 27px and the text in other table cells takes up only 22px (due to line-height: 22px). Just by adding cellClassName: 'iui-tables-slot' seems to solve that problem since it removes the paddings from that particular cell. I see the same strategy used for populating checkbox columns.